### PR TITLE
Add +++ and --- with right associativity to the list of custom operators

### PR DESCRIPTION
**Pull request by IvanRublev**

### CODE REVIEW

File: lib/elixir/lib/code/identifier.ex [hunk 1]
```diff
@@ -43,6 +43,7 @@ defmodule Code.Identifier do
       op in [:<, :<=, :>=, :>] -> {:left, 160}
       op in [:|>, :<<<, :>>>, :<~, :~>, :<<~, :~>>, :<~>, :<|>] -> {:left, 170}
       op in [:in] -> {:left, 180}
+      op in [:+++, :---] -> {:right, 185}
```
Comment (josevalim) [inline lib/elixir/lib/code/identifier.ex:46]: Can you please add it to the same list as `++` and `--`? Thanks.

**Comment by josevalim** [comment]
Hi @IvanRublev, thank you for the PR. It looks great! My only remark is to add this to the same list as `++/--/../<>`. Given they are currently called `two_op`, we can rename it to `concat_op` for now. Thanks!

**Comment by ericmj** [comment]
Thank you! 💜

**Comment by IvanRublev** [comment]
Folks, is it a ridiculous idea to include this one into Elixir release candidate v1.11? 🙂

**Comment by josevalim** [comment]
master=Elixir v1.11, so it should be there.

**Comment by josevalim** [comment]
Hi @IvanRublev! I am considering deprecating these two operators in favor of `<+>`, `<->`, `<*>`, `</>`, and `<**>`, that follow the conventional mathematical precedence. Please let me know if the new operators will also suffice for your use cases.

**Comment by josevalim** [comment]
Can you please explain why they have to be right-associative?

**Comment by IvanRublev** [comment]
The original use case was to have an right-associative operator to add a tag chain to a value like the following:
https://hexdocs.pm/tagged_tuple/TaggedTuple.html#---/2

**Comment by josevalim** [comment]
I see. So we will have to either keep them or introduce a new one. Thanks for clarifying!

**Pull request merged**